vendor/engines/secure_rooms/spec/system/admin_edit_occupancy_spec.rb

```diff
@@ -34,7 +34,7 @@
     let(:entry_at) { Time.zone.parse("2017-09-12 08:30") }
     let(:exit_at) { Time.zone.parse("2017-09-12 09:30") }
 
-    it "can change the times" do
+    it "can change the times", :blank_price_change_reason_options do
```

Comment:
This change is in OSU, but I'd guess we don't want it here, since if we ever have price change reason options, this test will ignore them.

Reply:
Good point, let's take that out.